src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -415,7 +416,6 @@ fn make_data(
         "description".to_owned(),
         json!(config.book.description.clone().unwrap_or_default()),
     );
-    data.insert("favicon".to_owned(), json!("favicon.png"));
```

Comment:
Shouldn't this also somehow be replaced?

Reply:
In `index.hbs` I replaced `{{ favicon }}` with just the literal text `favicon.png` and `favicon.svg`.  There didn't seem to be a reason to have this because it is a fixed value and is not configurable.